### ISSUE TITLE
Youtube music Login

### ISSUE THREAD

**Issue by ghalibansari** [Sangwan5688/BlackHole#498]
Please add support for youtube music login.

**Comment by Rafee-M** [comment]
Won't be added due to privacy concerns and Google plugins are closed source

**Comment by ghalibansari** [comment]
@Rafee-M Yeah, I understand, but then how these guys are doing [uYouPlus](https://github.com/qnblackcat/uYouPlus). thanks for making this awesome app.
